### PR TITLE
#126 - Apply Data Seed using Hasura Migration

### LINKED ISSUE #126: Seed Hasura db in a separate step
## Problem
- Hasura seed apply is its own container right now, with a race condition bug (discussed)
    - Long term, having a separate container script is not idempotent. 
    - Every time I start up docker compose, I re run the seed container, which is wasteful.
- Let's be specific about the purpose of this step
  - Naming is confusing, what are we actually seeding?
  - Where is the seed data?

## Ideal Flow
- Seed data should be in a seed.yaml equivalent somewhere
- Ideal flow
  1. docker compose up
  2. docker exec into the hasura container (is this step necessary?)
  3. run `seed.sh seed_data.yaml`

### PR DESCRIPTION
**Pull request by louis-jan**
- Hasura seed apply has its own container, which is currently inefficient.
- Apply data migration instead of data seed

fixes #126

### CODE REVIEW

File: conf/db/docker_psql_init.sql [hunk 1]
```diff
@@ -23,4 +24,42 @@ CREATE DATABASE "jan-hasura-metadata"
...
+ON CONFLICT (slug) DO NOTHING;
+
+-- INSERT COLLECTIONS DATA
+INSERT INTO public.collections ("slug", "name", "description") VALUES
+('conversational', 'Conversational', 'Chatbot alternatives to ChatGPT. Converse with these models and get answers.')
```
Comment (0xSage) [inline conf/db/docker_psql_init.sql:37]: Nit: remove `Chatbot alternatives to ChatGPT` since we do plan to be openai compatible?

**Review by 0xSage — APPROVED** [review]

File: conf/db/docker_psql_init.sql (hunk 1 above)
Comment (louis-jan) [reply conf/db/docker_psql_init.sql:37]: yay

**Comment by louis-jan** [comment]
Hey @0xSage I have updated the PR with a new approach. Would be better, please help take a look.

File: conf/db/docker_psql_init.sql (hunk 1 above)
Comment (louis-jan) [reply conf/db/docker_psql_init.sql]: Removed

**Review by hiento09 — APPROVED** [review]

**Pull request merged**